backend/arm64/emit.mlp

```diff
@@ -974,12 +975,18 @@ let emit_instr i =
     | Lop (Iprobe _ | Iprobe_is_enabled _) ->
         fatal_error ("Probes not supported.")
     | Lop (Icsel tst) ->
+      let len = Array.length i.arg in
+      let ifso = i.arg.(len - 2) in
+      let ifnot = i.arg.(len - 1) in
+      if Reg.same_loc ifso ifnot then
```

Comment:
I think it is OK, like the other instances,
because typing ensures that `ifso` and
`ifnot` are in the same register class.
Do you concur?

Reply:
Yes, that's why I didn't check register class.